docker-compose.yml

```diff
@@ -312,6 +329,10 @@ services:
       - app
 
 networks:
+  resources:
+    ipam:
+      config:
+        - subnet: 172.172.0.0/16
```

Comment:
```suggestion
        - subnet: 172.17.0.0/16
```

Reply:
Gabi just copied it from a sample "CIDR resource" in seeds. This address is internal to the Docker network and should not conflict with anything.

Reply:
@conectado do you want to change it? Changing address in seeds won't break anything.

Reply:
will go with `172.20.0.0/16` since using `172.17.0.0/16` often leads to an overlap with the default docker network